### DISCUSSION TITLE
Could not create new account from StoreFront

### DISCUSSION THREAD

**Discussion by oryany12** [saleor/saleor#9518] [Q&A]
I installed saleor according to this [guide](https://docs.saleor.io/docs/3.x/developer/installation).
then I went to http://localhost:3001/ (storefront), then tried to register a new account, when I pressed 'Register' button ,there was no response from the website.
I want to check this in local machine

There is any solution for this issue?

**Comment by karolkielecki** [comment]
Hi @oryany12, could you please share some screenshots? Do you receive any errors?

**Reply by oryany12** [reply]
![image](https://user-images.githubusercontent.com/83392258/162754033-5f5bf6e7-85a8-4fce-9161-b8810813f374.png)
Clicking on 'Register' make no traffic with the server and there is no changes in the DB (also no errors appeared)
